cli/bench/main.rs

```diff
@@ -244,18 +245,18 @@ fn get_binary_sizes(target_dir: &PathBuf) -> Result<Value> {
     }
 
     mtimes.insert(filename.clone(), file_mtime);
-    sizes.insert(filename, Value::Number(Number::from(meta.len())));
+    sizes.insert(filename, meta.len());
   }
 
-  Ok(Value::Object(sizes))
+  Ok(sizes)
 }
 
 const BUNDLES: &[(&str, &str)] = &[
   ("file_server", "./std/http/file_server.ts"),
   ("gist", "./std/examples/gist.ts"),
 ];
-fn bundle_benchmark(deno_exe: &PathBuf) -> Result<Value> {
-  let mut sizes = Map::new();
+fn bundle_benchmark(deno_exe: &PathBuf) -> Result<HashMap<String, u64>> {
+  let mut sizes: HashMap<String, u64> = HashMap::new();
```

Comment:
This can also be written like below. This is a little less verbose IMO.

```suggestion
  let mut sizes = HashMap::<String, u64>::new();
```